src/FSharp.Editing/ProjectSystem/AssemblyContentProvider.fs

```diff
@@ -47,8 +49,13 @@ type AnalyzerAssemblyLoader() as self =
 
     member __.AddDependencyLocation (_fullPath: string): unit = ()
 
-    member __.LoadFromPath(fullPath: string): System.Reflection.Assembly =
-        Assembly.Load (AssemblyName.GetAssemblyName fullPath)
+    member __.LoadFromPath (fullPath: string) =
+    #if !NETCORE
```

Comment:
Something really weird was going on here, using the proper the netcore apis calls  were making the build fail, and since Ionide's netcore tooling is borked there wasn't any easy way to get it to what it's supposed to be. It'll have to be resolved once everyone else has gotten their act together